llvm/test/CodeGen/AIE/aie2/schedule/postpipeliner/bitwisexor.mir

```diff
@@ -0,0 +1,163 @@
+    %out.addr.015 = phi ptr addrspace(7) [ %incdec.ptr, %for.body ], [ %out, %for.body.preheader ]
+    %in2.addr.014 = phi ptr addrspace(6) [ %incdec.ptr6, %for.body ], [ %in2, %for.body.preheader ]
+    %0 = load <64 x i8>, ptr addrspace(5) %in1.addr.016, align 32, !tbaa !2
+    %1 = load <64 x i8>, ptr addrspace(6) %in2.addr.014, align 32, !tbaa !2
+    %2 = tail call { <64 x i8>, <2 x i32> } @llvm.aie2.vbneg.ltz8(<64 x i8> %1)
+    %3 = extractvalue { <64 x i8>, <2 x i32> } %2, 0
+    %and.i = and <64 x i8> %3, %0
+    %4 = tail call { <64 x i8>, <2 x i32> } @llvm.aie2.vbneg.ltz8(<64 x i8> %0)
+    %5 = extractvalue { <64 x i8>, <2 x i32> } %4, 0
+    %and.i12 = and <64 x i8> %5, %1
+    %or.i = or <64 x i8> %and.i12, %and.i
+    %incdec.ptr = getelementptr inbounds i8, ptr addrspace(7) %out.addr.015, i20 64
+    store <64 x i8> %or.i, ptr addrspace(7) %out.addr.015, align 32, !tbaa !2
+    %incdec.ptr5 = getelementptr inbounds i8, ptr addrspace(5) %in1.addr.016, i20 64
+    %incdec.ptr6 = getelementptr inbounds i8, ptr addrspace(6) %in2.addr.014, i20 64
+    %6 = call i1 @llvm.loop.decrement.i32(i32 1)
+    br i1 %6, label %for.body, label %for.cond.cleanup, !llvm.loop !5
+
+  for.cond.cleanup:                                 ; preds = %for.body, %entry
+    ret void
+  }
+
+  declare { <64 x i8>, <2 x i32> } @llvm.aie2.vbneg.ltz8(<64 x i8>) #1
+  declare void @llvm.set.loop.iterations.i32(i32) #2
+  declare i1 @llvm.loop.decrement.i32(i32) #2
+
+  attributes #0 = { mustprogress nofree norecurse nosync nounwind memory(readwrite, inaccessiblemem: none) "no-trapping-math"="true" "stack-protector-buffer-size"="8" }
+  attributes #1 = { nocallback nofree nosync nounwind willreturn memory(none) }
+  attributes #2 = { nocallback noduplicate nofree nosync nounwind willreturn }
+
+  !llvm.module.flags = !{!0}
+  !llvm.ident = !{!1}
+
+  !0 = !{i32 1, !"wchar_size", i32 4}
+  !1 = !{!"clang version 19.0.0git (git@github.com:Xilinx/llvm-aie.git 7dacd4f131902667e5df5f8a9eb9322a88a81437)"}
+  !2 = !{!3, !3, i64 0}
+  !3 = !{!"omnipotent char", !4, i64 0}
+  !4 = !{!"Simple C++ TBAA"}
+  !5 = distinct !{!5, !6, !7}
+  !6 = !{!"llvm.loop.mustprogress"}
+  !7 = !{!"llvm.loop.itercount.range", i64 10}
+
+...
+---
+name:            bitXor
+alignment:       16
+tracksRegLiveness: true
+body:             |
+  bb.0.entry (align 16):
+    successors: %bb.1(0x50000000), %bb.3(0x30000000)
+    liveins: $p0, $p1, $p2, $r0
+
+    $r1 = MOV_RLC_imm10_pseudo 0
+    $r1 = GE $r1, $r0
+    JNZ $r1, %bb.3
+    DelayedSchedBarrier
+
+  bb.1.for.body.preheader:
+    successors: %bb.2(0x80000000)
+    liveins: $p0, $p1, $p2, $r0
+
+    $lc = ADD_NC $r0, 0
+    $ls = MOVXM_lng_cg %bb.2
+    $le = MOVXM_lng_cg <mcsymbol .L_LEnd0>
+
+  bb.2.for.body (align 16):
+    successors: %bb.2(0x7c000000), %bb.3(0x04000000)
+    liveins: $p0, $p1, $p2
+
+    $wh0 = VLD_idx_imm_3x32_pseudo $p1, 32 :: (load (<32 x s8>) from %ir.in2.addr.014 + 32, !tbaa !2, addrspace 6)
```

Comment:
Nit: I'd prefer to use "fake" MMOs like `load from stack - 32` if it means we can get rid of the IR and debloat the file :)